### ISSUE TITLE
Dataset config for training.

### ISSUE THREAD

**Issue by adityarp9** [PiotrDabkowski/pytorch-saliency#2]
For training your model, we need  IMAGENET 2015 dataset for the object classification/localization right? Well, in that dataset the training set is sorted into folders according to class. However, the validation set is not sorted in folders i.e. all 50000 images are in a single folder. Your training code isn't accepting that.

As a  hack, I put all the 50k validation images in 1 subfolder (redundant). Training has started, but will the cross-validation be screwed up since I haven't sorted val images according to the classes?
 Please suggest a solution? Any help is much appreciated.

**Comment by PiotrDabkowski** [comment]
Yeah, there are 2 common formats for storing ImageNet validation set, I am using one that is compatible with the training set and that requires you to either write a short script yourself to organise the images into folders or just look for a script on the internet, I am sure you will find it.

**Comment by adityarp9** [comment]
Okay, cool. :+1:
